civis_jupyter_notebooks/assets/jupyter_notebook_config.py

```diff
@@ -25,6 +25,9 @@
 c.NotebookApp.open_browser = False
 c.NotebookApp.token = ''
 c.NotebookApp.tornado_settings = {'headers': {'Content-Security-Policy': "frame-ancestors *"}}
+c.NotebookApp.terminado_settings = {'shell_command': ['bash']}
+c.NotebookApp.allow_root = True
+c.FileContentsManager.post_save_hook = platform_persistence.post_save
```

Comment:
Are this line and the one above (`allow_root = True`) duplicating other lines in this file?